addons/mail/static/src/new/core/channel_member_model.js

```diff
@@ -5,10 +5,25 @@
  * @typedef Data
  * @property {number} id
  * @property {number} partnerId
+ * @property {number} threadId
  */
 export class ChannelMember {
-    constructor(_state, { id, partnerId }) {
-        Object.assign(this, { _state, id, partnerId });
+    static insert(state, data) {
+        let channelMember = state.channelMembers[data.id];
+        if (!channelMember) {
+            channelMember = new ChannelMember();
+            channelMember._state = state;
+            state.channelMembers[data.id] = channelMember;
```

Comment:
it still doesn't return a reactive

Reply:
Oops forgot this one. Will either make a fix soon, or eventually @oomsveta implement solution for all models once and for all.